From af9fd4bad4aa3e4358d86138f9fc632baeb0fdeb Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Mon, 16 Apr 2018 14:25:24 +0200
Subject: [PATCH] Don't need to run previous iterations commands again

Summary: BUG: 393141

Test Plan:
Tested building plasma-workspace, it was clearly faster (although it
would have finished eventually)

Reviewers: #frameworks, arojas

Subscribers: heikobecker

Tags: #frameworks

Differential Revision: https://phabricator.kde.org/D12216
---
 cmake/build-pofiles.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/build-pofiles.cmake b/cmake/build-pofiles.cmake
index d0991ad..b39be31 100644
--- a/cmake/build-pofiles.cmake
+++ b/cmake/build-pofiles.cmake
@@ -62,6 +62,7 @@ foreach(pofile IN LISTS pofiles)
     if(i EQUAL ${numberOfProcesses})
         _processCommands()
         set(i 0)
+        set(commands)
     endif()
 endforeach()

--
2.16.2.windows.1

